helpdesk_mgmt/security/helpdesk_security.xml

```diff
@@ -26,7 +26,7 @@
 <data noupdate="1">
 
 
-    <record id="helpesk_ticket_personal_rule" model="ir.rule">
+    <record id="helpdesk_ticket_personal_rule" model="ir.rule">
```

Comment:
You can't rename a noupdate="1" record, or you will have 2 records when updating. We have to stick with this bad name (but not really a problem), or do a migration script renaming the record.

Reply:
Thats true:) i removed the change